src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsaccreditedprogrammesapi/restapi/config/ApiExceptionHandler.kt

```diff
@@ -47,6 +49,26 @@ class RestConfiguration {
       )
   }
 
+  @ExceptionHandler(ServiceUnavailableException::class)
+  fun handleServiceUnavailableException(e: ServiceUnavailableException): ResponseEntity<ErrorResponse> {
+    log.error("Service unavailable ${e.message}", e)
+
+    val serviceMessage = when (e) {
+      is PrisonApiUnavailableException -> "Prison api is not available at the moment, please try after sometime"
```

Comment:
Can we make this "Prison API is temporarily unavailable. Please try again later."